portal/cookbook-guide.md

```diff
@@ -6,84 +6,261 @@ They are [geoscience](https://en.wikipedia.org/wiki/Earth_science)-focused
-
-## Set up the environment
-
-1. [Clone the repository](https://foundations.projectpythia.org/foundations/github/github-cloning-forking.html) in your local workspace
-1. Within `environment.yml`, change `name` from `cookbook-dev` to `<your-cookbook-name>-dev` (e.g. `cesm-cookbook-dev`) and add all required libraries and other dependencies under `dependencies:`. Commit the changes
-1. Create the Conda environment with `conda env create -f environment.yml`. If it crashes, try running `conda config --set channel_priority strict`
+1. Under Settings &rarr; Pages, ensure that GitHub Pages is enabled. If it is not, change the Branch from "None" to "gh-pages/(root)" and click "Save"
+1. Under Settings &rarr; Actions &rarr; General, allow Github Actions to **push** to the repository <img width="901" alt="Screenshot 2023-01-13 at 3 12 47 PM" src="https://user-images.githubusercontent.com/26660300/212428991-cd0ae2f0-73ca-40d8-b983-f122359463aa.png">
+
+Your cookbook is now ready to have content added!
+
+```{Note}
+In the rest of this guide, we assume that you are familiar with the basics of using git and GitHub. If not, we strongly recommend reading through our [GitHub tutorials in Foundations](https://foundations.projectpythia.org/foundations/getting-started-github.html).
+```
+
+---
+
+## Transfer your cookbook repo to the ProjectPythia organization
+
+In order for your Cookbook to be included in the Gallery, the source repository needs to be housed within the [Project Pythia GitHub organization](https://github.com/ProjectPythia).
+
+You can keep your repository in your personal GitHub space while you're developing your content if that works better for you. Repository ownership can be transfered at any time.
+
+However, we recommend transfering to the Pythia organization early, for a few reasons:
+- Fewer settings to tweak later
+- Easier to get help from the Pythia infrastructure team
+- Encourages collaboration
+
+```{Note}
+You're still in control! You will always retain write access to your Cookbook repository even after transfering ownership to the Pythia organization.
+
+Also, _don't worry about breaking anything!_ Your repo will not affect any other Project Pythia content until you initiate the request to list it on the [Cookbook Gallery](https://cookbooks.projectpythia.org) (see below...)
+```
+
+### Steps to transfer the repository
+
+1. [Contact the Pythia team via the Pangeo Discourse](https://discourse.pangeo.io/c/education/project-pythia/60) (or otherwise) to let us know about your cookbook.
+1. You will get an invitation to join the the [ProjectPythia organization](https://github.com/ProjectPythia).
+1. Accept the GitHub invitation. _Welcome to the team!_
+1. Once you are a member of the organization, navigate to the Settings of your cookbook repository and scroll down to the **Danger Zone**.
+1. Click "Transfer".
+1. Select or type "ProjectPythia", confirm, and transfer.
```

Comment:
There is another step following this one that asks which teams to give access to. Options are nobody, core, education, infrastructure, outreach, and "find a team". What is the guidance here?

Reply:
Hmm I guess the guidance should be to select "core". I think that includes all the active members of both "education" and "infrastructure". We haven't been very careful or consistent about how we use those teams.

Reply:
I selected "core" when I transferred a [new cookbook repo](https://github.com/ProjectPythia/radiative-feedback-cookbook) the other day, which is what got me started on this guide revision.

I forgot to include this guidance in the text.

Reply:
Resolved: https://projectpythia.org/_preview/435/cookbook-guide.html#steps-to-transfer-the-repository